src/OrbitGl/CallTreeView.cpp

```diff
@@ -118,14 +120,19 @@ uint64_t CallTreeNode::GetExclusiveSampleCount() const {
 static void AddCallstackToTopDownThread(CallTreeThread* thread_node,
                                         const CallstackInfo& resolved_callstack,
                                         uint64_t callstack_sample_count,
+                                        const ModuleManager* module_manager,
```

Comment:
Can this be a const reference? Same over the entire file.

Reply:
Good catch. Done.